### PR TITLE
Auto prefixer (PostCSS modules)

### PR DESCRIPTION
**Pull request by JakeSidSmith**
To be merged #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,10 +61,23 @@ You can define as many files as you like, as long as there is an output argument
...
 
-### Options
+### PostCSS
+
+You can use PostCSS modules with watch-less-do-more incredibly easily.
+Just install the modules you want, and tell watch-less-do-more to use them in the same way you would with PostCSS.
+
+```json
+{
+  "watch-less": "watch-less-do-more -u autoprefixer -i source.less -o output.css"
```
Comment (RealOrangeOne) [inline README.md]: it'd be nice if rather than showing the entire command, you just show the options that need to be added

Comment (JakeSidSmith) [inline README.md]: See the options below. 😉

Comment (RealOrangeOne) [inline README.md]: In that case, just drop the example here

Comment (JakeSidSmith) [inline README.md]: I'm highlighting a feature, not documenting the API.

**Pull request merged**